src/print.rs

```diff
@@ -426,9 +436,13 @@ fn load_formatter_and_modules<'a>(context: &'a Context) -> (StringFormatter<'a>,
     if cformatter.is_err() {
         log::error!("Error parsing `continuation_prompt`")
     }
-
-    match (lformatter, rformatter, cformatter) {
-        (Ok(lf), Ok(rf), Ok(cf)) => {
+    if pformatter.is_err() {
+        if let Some(profile) = profile_name {
+            log::error!("Error parsing profile `{}`", profile)
+        }
+    }
+    match (lformatter, rformatter, cformatter, pformatter) {
+        (Ok(lf), Ok(rf), Ok(cf), Ok(pf)) => {
             let mut modules: BTreeSet<String> = BTreeSet::new();
             if context.target != Target::Continuation {
                 modules.extend(lf.get_variables());
```

Comment:
This should either be limited to `matches!(context.target, Target::Main | Target::Right)` instead. Alternatively, this should maybe also handle `Profile(_)` and add either `lf` or `pf` depending on which is active to `modules` allow both to ignore `rf` modules from `$all`.

Reply:
If I understand correctly, we initialize either the Main OR Profile(_) + Right and pass on modules both. Is that right?

Reply:
Yes.

Reply:
@davidkna just updated the code, let me know what you think.